### DISCUSSION TITLE
Support for app open ad unit

### DISCUSSION THREAD

**Discussion by Pl0414141** [admob-plus/admob-plus#313] [Fund Features]
See https://support.google.com/admob/answer/9620632?hl=en and https://developers.google.com/admob/android/app-open-ads

**Comment by victorvhpg** [comment]
any update?
https://developers.google.com/admob/android/app-open-ads
we need that support

**Comment by mcfarljw** [comment]
We are also interested in support for the new ad format.

**Reply by EMI-INDO** [reply]
> Kami juga tertarik dengan dukungan untuk format iklan baru.

Here  https://github.com/admob-plus/admob-plus/discussions/314

**Comment by eliadAfeka** [comment]
@ratson can you open kind of a place where we can donate and see how much we got ? so we all donate as much as we can until you will arrive at the goal (1000$) 
we also need AD OPEN and NATIVE AD

**Reply by ratson** [reply]
@eliadAfeka Added progress bar in https://github.com/admob-plus/admob-plus/discussions/314

**Comment by creakosta** [comment]
@ratson ready pay for this $150

**Comment by eliadAfeka** [comment]
@ratson 150$

**Comment by eliadAfeka** [comment]
@ratson how much is missing to complete?

**Comment by ratson** [comment]
@victorvhpg @mcfarljw @creakosta @eliadAfeka @EMI-INDO @Crasheros @krunalsk007
I am going to start working on this feature, you may send your fund via
- https://github.com/sponsors/ratson?frequency=one-time
- https://www.paypal.com/paypalme/ratsonal

![](https://progress-bar.dev/35)

**Reply by creakosta** [reply]
Done

**Reply by mcfarljw** [reply]
We have chipped in as well

**Reply by eliadAfeka** [reply]
done. (@ratson I will complete what's missing in the next days if it will not arrived to your goal)

**Reply by EYALIN** [reply]
@ratson I added 400 more in addition to the 150 I already gave. (from @eliadAfeka )

**Comment by ratson** [comment]
I have just released `admob-plus-cordova@1.13.0` with this feature.
The documentation is https://admob-plus.github.io/docs/cordova/ads/app-open

**Reply by EYALIN** [reply]
@ratson I will test it now. 
can you please let me know how much missing for your goal?
the bar in the link showing how many sponsors you have.  
(eliadafeka)

**Reply by ratson** [reply]
@EYALIN Still need $650 to reach the goal of this feature.

**Comment by sksk008** [comment]
how to implement this feature in ionic  ?

**Reply by EMI-INDO** [reply]
> how to implement this feature in ionic ?

https://admob-plus.github.io/docs/ionic
